### PR TITLE
Adds PagerDuty CLI support

### PR DESCRIPTION
**Pull request by clcollins**
~Installs `go-pagerduty` [https://github.com/PagerDuty/go-pagerduty](https://github.com/PagerDuty/go-pagerduty), and adds the necessary mount for the authentication token.~

Installs `pagerduty-cli` [https://github.com/martindstone/pagerduty-cli](https://github.com/martindstone/pagerduty-cli), and adds the necessary mount for the authentication token.

Signed-off-by: Christopher Collins <collins.christopher@gmail.com>

### CODE REVIEW

**Comment by georgettica** [comment]
Is go-pagerduty better for you than https://github.com/martindstone/pagerduty-cli?

If not I am a maintainer in go-pagerduty and will support any changes there to make it better

**Comment by clcollins** [comment]
@georgettica: TBH, the cli usage for `go-pagerduty` could use some work to make it a bit more friendly, but I was leaning toward it over `pagerduty-cli` because I don't know/trust the developer, i'm a little creeped out by the installer, and adding npm to install directly adds 150M to ocm-container.  It is WAY more user-friendly, though.  I'm torn, and would be happy to discuss.

**Comment by clcollins** [comment]
@georgettica: After further investigation, as size was the only issue, replaced `go-pagerduty` with `pagerduty-cli` in this PR.  Certainly WAY better UI.

File: Dockerfile [hunk 1]
```diff
@@ -174,6 +173,9 @@ COPY utils/bin /root/.local/bin
 # Setup requirements for cluster-login.sh and install o-must-gather
 RUN pip3 install requests-html o-must-gather && pyppeteer-install
 
+# Setup pagerduty-cli
+RUN npm install -g pagerduty-cli
```
Comment (dustman9000) [inline Dockerfile:177]: We should probably lock this down to a specific version or use a package.json

Comment (clcollins) [reply Dockerfile:177]: Added tag for specific version: `RUN npm install -q pagerduty-cli@0.0.70`

**Comment by georgettica** [comment]
The maintainer is someone from pagerduty @mmazur can attest to that

**Pull request merged**